app/src/main/kotlin/com/wire/android/util/CamelCase.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.android.util
+
+@Suppress("FunctionNaming")
+fun String.ToCamelCase(delimiter: String = " ", separator: String = " "): String =
```

Comment:
Why uppercase `To` ?

```kotlin
val someString = "foo bar"
someString.ToCamelCase()?
```

Reply:
Also, it looks like a title case, not a camel case.

`camelCase` starts with lowercase, just like function names.

Reply:
Also... missing tests.
This one should be a fun. No dependencies.